code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -379,7 +379,7 @@ var/bomb_set
 		ticker.mode.station_was_nuked = (off_station<2)	//offstation==1 is a draw. the station becomes irradiated and needs to be evacuated.
 														//kinda shit but I couldn't  get permission to do what I wanted to do.
 		if(!ticker.mode.check_finished())//If the mode does not deal with the nuke going off so just reboot because everyone is stuck as is
-			world.Reboot("Station destroyed by Nuclear Device.", "end_error", "nuke - unhandled ending")
+			spawn() world.Reboot("Station destroyed by Nuclear Device.", "end_error", "nuke - unhandled ending")

```

Comment:
```
game/gamemodes/meteor/meteor.dm:24:	spawn() spawn_meteors(6, meteors_normal)
game/machinery/magnet.dm:347:					spawn() MagnetMove()
game/machinery/syndicatebeacon.dm:101:	spawn() explosion(src.loc, rand(3,8), rand(1,3), 1, 10)
modules/admin/verbs/SDQL2/SDQL_2.dm:99:								spawn() call(d, v)(arglist(args_list)) // Spawn in case the function sleeps.
modules/library/lib_machines.dm:441:				spawn() src.Topic(nhref, params2list(nhref), src)
modules/mob/login.dm:17:					spawn() alert("You have logged in already with another key this round, please log out of this one NOW or risk being banned!")
```